### DISCUSSION TITLE
Installation path on Apple M1 Silicon

### DISCUSSION THREAD

**Discussion by keiwanjamaly** [Homebrew/discussions#2676] [Casks]
Hello folks,

So reading the [FAQs](https://docs.brew.sh/FAQ#why-is-the-default-installation-prefix-opthomebrew-on-apple-silicon) the default install location on Silicon is `/opt/homebrew`. I've installed GLFW with brew and use cmake [FindGLW3](https://github.com/JoeyDeVries/LearnOpenGL/blob/master/cmake/modules/FindGLFW3.cmake) to look it up. The problem I encounter is, that it does not find this package, since it is not located in the default search path (`/usr/local/include` is in the search path). So is there a general workaround for this? Because, I expect, that this will occur multiple times somewhere?

**Comment by SMillerDev** [comment]
It seems like you can define your own location using `GLFW3_ROOT` with that module

**Reply by keiwanjamaly** [reply]
I know, but then I have to set the Flag 

--help-policy CMP0074

and this will also break compatibility to Linux.
